dist/index.html

```diff
@@ -28,79 +28,77 @@
+            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Critique Standards</h3>
+            <p class="main-program-grid__grid-item__content lato_regular">We are here to critique shows, not critize them, so it's our goal to keep all the haters out and only accept criticism based on facts</p>
           </div>
           <div class="main-program-grid__grid-item">
             <i class="fa-solid fa-comments fa-2x"></i>
-            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Forum</h3>
-            <p class="main-program-grid__grid-item__content lato_regular">Have the time to share your thoughts and opinions with experts for each topic.</p>
+            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Discussion</h3>
+            <p class="main-program-grid__grid-item__content lato_regular">Nobody owns the truth, so to reach a general consensus on wether a show is good or bad its necessary to discuss stuff among the community, we will enable Disqus as soon as possible</p>
           </div>
           <div class="main-program-grid__grid-item">
             <i class="fa-solid fa-laptop fa-2x"></i>
-            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Workshop</h3>
-            <p class="main-program-grid__grid-item__content lato_regular">Try creating your own work using open source license rather than just watching at it.</p>
+            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Watch more!</h3>
+            <p class="main-program-grid__grid-item__content lato_regular">We want our critics to be knowledgeable, so for this cause, we are only allowing people with more than a hundred shows watched publish critiques, feel free to discuss though!</p>
           </div>
           <div class="main-program-grid__grid-item">
             <i class="fa-brands fa-connectdevelop fa-2x"></i>
-            <h3 class="main-program-grid__grid-item__title cocogoose_regular">CC Ignite</h3>
-            <p class="main-program-grid__grid-item__content lato_regular">Get opportunities to network with CC affiliates around the world, aslo after the summit.</p>
+            <h3 class="main-program-grid__grid-item__title cocogoose_regular">Network</h3>
+            <p class="main-program-grid__grid-item__content lato_regular">Get the opportunity to make new acquaintances and friends from all over the world, share and discuss shows, but most importantly, HAVE FUN!</p>
           </div>
-          <div class="main-program-grid__grid-item__button"><a href="#">Join CC Global Summit 2015</a></div>
+          <div class="main-program-grid__grid-item__button"><a href="#">Join Animation Fans as a Critic</a></div>
         </div>
-        <a href="#" class="main-program__see-more lato_regular">SEE THE WHOLE PROGRAM</a>
+        <a href="#" class="main-program__see-more lato_regular">SEE THE WHOLE GOALS LIST</a>
       </section>
       <section class="featured lato_regular">
```

Comment:
- [ ] This section is made up of six individual articles.

- [ ] For the speaker section, the names of the speakers are aligned with the top part of the images, while for this project the names are slightly over the top of the images, kindly align them.

Provided Design
![image](https://user-images.githubusercontent.com/81932478/169597479-6f617351-08c8-42bb-a642-afdc49c668e5.png)

Your Design
![image](https://user-images.githubusercontent.com/81932478/169597593-eb0757ce-4846-4b65-898c-a09b51b4c6cd.png)

Reply:
Successfully aligned. 
Thanks